File: GTTS/INL.TrainingService.Models/TrainingEventTypesAtBusinessUnit_Item.cs

namespace INL.TrainingService.Models
{
    public class TrainingEventTypesAtBusinessUnit_Item : ITrainingEventTypesAtBusinessUnit_Item
    {
        public int TrainingEventTypeID { get; set; }
        public string TrainingEventTypeName { get; set; }
        public long BusinessUnitID { get; set; }
        public string Acronym { get; set; }
        public string BusinessUnitName { get; set; }
        public bool BusinessUnitActive { get; set; }
        public bool TrainingEventTypeBusinessUnitActive { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/training-event-types-at-business-unit_item.ts


import { ITrainingEventTypesAtBusinessUnit_Item } from './itraining-event-types-at-business-unit_item';

export class TrainingEventTypesAtBusinessUnit_Item implements ITrainingEventTypesAtBusinessUnit_Item {
  
	public TrainingEventTypeID: number = 0;
	public TrainingEventTypeName: string = "";
	public BusinessUnitID: number = 0;
	public Acronym: string = "";
	public BusinessUnitName: string = "";
	public BusinessUnitActive: boolean = false;
	public TrainingEventTypeBusinessUnitActive: boolean = false;
  
}






